.travis.yml

```diff
@@ -11,98 +11,98 @@ matrix:
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-rpc-state-frontier"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-native-state-homestead"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-rpc-state-homestead"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-native-state-eip150"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-rpc-state-eip150"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-native-state-eip158"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-rpc-state-eip158"
-    #- python: "3.6"
-    #- env: TOX_POSARGS="-e py36-native-state-constantinople"
-    #- python: "3.6"
-    #- env: TOX_POSARGS="-e py36-native-state-metropolis"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-native-blockchain"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-rpc-blockchain"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-vm"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-core"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-transactions"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-p2p"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-database"
-    - python: "3.6"
-      env: TOX_POSARGS="-e py36-rpc-state-quadratic"
-    #- python: "3.6"
-    #- env: TOX_POSARGS="-e py36-vm-performance"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-native-state-byzantium"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-state-byzantium"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-native-state-frontier"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-state-frontier"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-native-state-homestead"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-state-homestead"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-native-state-eip150"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-state-eip150"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-native-state-eip158"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-state-eip158"
+    #-# -  python: "3.6"
+    #-# -  env: TOX_POSARGS="-e py36-native-state-constantinople"
+    #-# -  python: "3.6"
+    #-# -  env: TOX_POSARGS="-e py36-native-state-metropolis"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-native-blockchain"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-blockchain"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-vm"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-core"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-transactions"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-p2p"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-database"
+    - # - python: "3.6"
+      # - env: TOX_POSARGS="-e py36-rpc-state-quadratic"
+    #-# -  python: "3.6"
+    #-# -  env: TOX_POSARGS="-e py36-vm-performance"
```

Comment:
Hah, that's one way to easily pass the tests. Just a reminder to revert this before merging...

Side thought: maybe we could only run the py36 tests in the cron job? Which reminds me that that is still probably broken from long-running tests that I said I would clean up. Adding a note to myself...